tests/datatypes/test_list.py

```diff
@@ -58,10 +58,6 @@ class UnaryListOperationTests(UnaryOperationTestCase, TranspileTestCase):
     values = ['[]', '[1, 2, 3, 4, 5]']
 
     not_implemented = [
-        'test_unary_positive',
-        'test_unary_negative',
-        'test_unary_not',
-        'test_unary_invert',
     ]
```

Comment:
If you remove the last of the non-implemented tests, you should be able to delete the not_implemented marker entirely (and if you can't, you should, and it's a bug). Once we've got a completely compliant implementation, there's no need for cruft to hang around telling us that there's nothing that isn't implemented.